be/src/io/fs/s3_file_system.cpp

```diff
@@ -141,7 +141,7 @@ Status S3FileSystem::open_file(const Path& path, FileReaderSPtr* reader) {
     RETURN_IF_ERROR(file_size(path, &fsize));
     auto key = get_key(path);
     auto fs_path = Path(_s3_conf.endpoint) / _s3_conf.bucket / key;
-    *reader = std::make_unique<S3FileReader>(std::move(fs_path), fsize, std::move(key),
+    *reader = std::make_shared<S3FileReader>(std::move(fs_path), fsize, std::move(key),
```

Comment:
Why change to shared ?

Reply:
This is a typo(not bug) fix, as function signature is `S3FileSystem::open_file(const Path& path, FileReaderSPtr* reader)`